### DISCUSSION TITLE
fluidflux coastline

### DISCUSSION THREAD

**Discussion by paulohenriquefurtadopereira** [Sartaq12/WaterInteractionPlugin#27] [Ideas]
congratulations on the work. It really is something incredible. But one question, would there be a way to make a fluidflux+ ninjalive plugin? because the waves on the fluisflux coast are perfect. or if it were possible, just integrate the fluidflux coasts into the current plugin to work in oceanology

**Comment by Sartaq12** [comment]
Integrating fluidflux into oceanology doesn't make much sense.
Technically you can use NinjaLive with fluidflux, but I haven't worked with it, so dunno what all the unknowns are.

**Comment by Tmllj** [comment]
Dude, the material function is broken and Oceanology_FluidNinja can't find the parent class? I have used Oceanology_plugin version 5.1, can you fix it

**Reply by Sartaq12** [reply]
Are you using the old release for 5.1? [v1.2.0-UE5.1](https://github.com/Sartaq12/WaterInteractionPlugin/releases/tag/v1.2.0-UE5.1)
Does this have anything to do with the question of this thread?
